docs-utils/update-images.js

```diff
@@ -27,27 +28,25 @@ const updateImages = async (filepath) => {
   const locale = frontmatter(content).attributes.locale
```

Comment:
The variables `locale` and `slug` are not used. Their declaration can be removed.

Reply:
Ok, removed.